testing/btest/broker/remote_event.zeek

```diff
@@ -1,3 +1,5 @@
+# @TEST-REQUIRES: test -x ${BUILD}/auxil/spicy/spicy/bin/spicy-config
+#
```

Comment:
Why do the Broker tests care about `spicy-config`?

Reply:
The output of Broker's `num_logs_outgoing` depends on whether Spicy is loaded or not (with Spicy loaded it is one larger). This appears to be deterministic, but looking over the implementation for that counter it is unclear to me why it would depend on Spicy, so I opted to disable the tests instead of digging further.